openlibrary/templates/work_search.html

```diff
@@ -278,16 +278,10 @@ <h3 class="booktitle">
                 </span>
             $elif doc.lending_edition and (library or 'inlibrary' not in doc.collections):
```

Comment:
Is doc a storage obj? Does lending_edition have a default value? We might want to getattr on it or doc.get('lending_edition')